docs/secure-connections/sauce-connect-5/installation.md

```diff
@@ -1,217 +1,12 @@
-rm sauce-connect_${sc_version}.linux_${arch}.deb
-```
-
-<details><summary>What does the Sauce Connect Debian package install</summary>
-
-- Sauce Connect Proxy binary is in `/usr/bin/sc`
-- The enviroment variables file template is in `/etc/default/sauce-connect`. The file may be modified to include your configuration, , see [Running systemd service on Debian-based Linux](/secure-connections/sauce-connect-5/operation/systemd/)
-  ```bash
-  cat /etc/default/sauce-connect
-  # Default values for Sauce Connect Proxy
-  #SAUCE_CONFIG_FILE=/etc/sauce-connect/config.yaml
-  # Required values
-  #SAUCE_USERNAME=
-  #SAUCE_ACCESS_KEY=
-  #SAUCE_REGION=
-  #SAUCE_TUNNEL_NAME=
-  # Options
-  #SAUCE_SHARED=all
-  #SAUCE_TUNNEL_POOL=
-  # See https://docs.saucelabs.com/dev/cli/sauce-connect-5/ for all environment variable values
-  ```
-- Systemd service is enabled, see [Running systemd service on Debian-based Linux](/secure-connections/sauce-connect-5/operation/systemd/)
-
-</details>
-
-#### RPM
-
-```bash title="Install an RPM package"
-sudo rpm -i https://saucelabs.com/downloads/sauce-connect/5.0.0/sauce-connect-5.0.0_linux.x86_64.rpm
-```
-
-  </TabItem>
-
-  <TabItem value="macOS">
-
-#### Homebrew
-
-```bash title="Homebrew (recommended)"
-brew install saucelabs/tap/sauce-connect
-```
-
-#### Binary
-
-```bash title="Download binary"
-mkdir $HOME/sauce-connect-5.0.0 && cd $HOME/sauce-connect-5.0.0
-curl -sLO "https://saucelabs.com/downloads/sauce-connect/5.0.0/sauce-connect-5.0.0_darwin.all.zip"
-unzip sauce-connect-5.0.0_darwin.all.zip && rm sauce-connect-5.0.0_darwin.all.zip
-```
-
-<details><summary>What's in the folder?</summary>
-
-#### Sauce Connect folder contents
-
-```bash
-  ├── LICENSE
-  ├── LICENSE.3RD_PARTY
-  ├── completions
-  │   ├── sc.bash
-  │   ├── sc.fish
-  │   └── sc.zsh
-  └── sc
-```
-
-</details>
-
-  </TabItem>
-
-  <TabItem value="Windows">
-
-```bash title="Using Powershell (Windows)"
-Invoke-RestMethod -Uri https://saucelabs.com/downloads/sauce-connect/5.0.0/sauce-connect-5.0.0_windows.x86_64.zip -OutFile sauce-connect-5.0.0.zip
-Expand-Archive -Force -Path ./sauce-connect-5.0.0.zip
-```
-
-  </TabItem>
-</Tabs>
-
-## Running Sauce Connect in Docker
-
-See [this guide](/secure-connections/sauce-connect-5/operation/docker/)
-
-## Running Sauce Connect in Kubernetes
-
-See [this guide](/secure-connections/sauce-connect-5/operation/kubernetes/)
-
-## More Information
-
-- [Sauce Connect Proxy Changelog](https://changelog.saucelabs.com/en?category=sauce%20connect)
-- [Sauce Connect Proxy 5 Administration](/secure-connections/sauce-connect-5/operation/overview/)
-- [Sauce Connect Proxy 5 CLI Reference](/dev/cli/sauce-connect-5/)
-- [Sauce Connect Proxy Architecture](/secure-connections/sauce-connect-5/advanced/architecture/)
+Visit the following pages for installation instructions for your platform:
+- [Docker](/secure-connections/sauce-connect-5/installation/docker)
```

Comment:
maybe the order should be OSes first and then other:
- linux
- mac
- win
- docker
- k8s
We could add plugins there too, later: github action, Jenkins, etc...

Reply:
ok